internal/graph/check_test.go

```diff
@@ -996,6 +996,75 @@ condition condition1(param1: string) {
 	require.False(t, resp.Allowed)
 }
 
+func TestCheckDispatchCount(t *testing.T) {
```

Comment:
Can we add a subtest that just ensures we don't start counting a direct lookup as a dispatch with a code change in the future?

e.g.

```
type user

type document
  relations
    define viewer: [user]
```

`Check(document:1#viewer@user:jon)` should never have a positive dispatch count. It should always be 0, but with a datastore query count of 1.